src/co/instea/BasicCommands_01.java

```diff
@@ -0,0 +1,24 @@
+package co.instea;
+
+import java.util.Scanner;
+
+public class BasicCommands_01 {
+    public static void main(String[] args) {
+        System.out.println("Enter following commands: ");
+        System.out.println("* 'h' for help");
+        System.out.println("* 'q' to quit the program");
+        Scanner input = new Scanner(System.in);
+        String letter = "";
+
+        //command line with neverending loop unless 'q' pressed for quit
+        while(true) {
+            letter = input.nextLine();
+            if(letter.equals("q"))
```

Comment:
when more commands are introduced, if/else if pattern can be cumbersome to use

Reply:
is a Switch pattern a better solution for this type of console app?

Reply:
switch is a little bit better, but I think we will introduce more sophisticated solution later - no need to change current approach